src/components/ThreadEnvelope.vue

```diff
@@ -375,9 +389,44 @@ export default {
 	}
 	.left {
 		flex-grow: 1;
+		//TODO: fix the important svg size
+		//hack to fix the important.svg overlapping the sender and subject.
+		z-index: 10000;
```

Comment:
@skjnldsv i need you help. For some reasons the svg of the important icon takes too much space. When i hover the thread, it is actually hovered the important icon. Can you please check whats wrong? i added z-index but doesn't really fix it.

Reply:
Might be capped by parent zIndex?